docs/config.md

```diff
@@ -28,3 +28,23 @@ with each sample having a shape of:
 $$(d_1, ... d_D)$$
 
 The option can be used to split batch of sequences into a batch of images or sub-sequences.
+
+## Backend parameters
+
+### `release_after_unload`
+
+One of the most expensive actions DALI takes while processing is the GPU memory allocation. DALI's
+memory ownership model assumes that the memory which is once allocated will be reused throughout
+the lifetime of the process. In other words, when DALI has allocated the memory, it is not released even
+after the DALI Pipeline object gets deleted. The release of the memory happens at the process quit [^1].
+
+This property entails that when using Triton, the GPU memory allocated by DALI is not freed, even
+after unloading the DALI model. However, should you like to free the GPU memory when DALI model is
+unloaded, please pass `release_after_unload` option to `tritonserver` invocation:
+
+```bash
+tritonserver --model-repository /models --log-verbose 1 --backend-config=dali,release_after_unload=true
+```
```

Comment:
I considered it and while this would be intuitive and I believe nice default behaviour, I'm worried about current use cases. There might be a situation where changing the default action will introduce unexpectedly high launching times. What do you think about this?